twill-yarn/src/main/java/org/apache/twill/internal/appmaster/ApplicationMasterService.java

```diff
@@ -221,7 +229,8 @@ protected void doStart() throws Exception {
 
     // initialize the event handler, if it fails, it will fail the application.
     if (eventHandler != null) {
-      eventHandler.initialize(new BasicEventHandlerContext(twillSpec.getEventHandler()));
+      eventHandler.initialize(new BasicEventHandlerContext(twillRuntimeSpec, twillSpec.getEventHandler()));
```

Comment:
The `twillRuntimeSpec` already contains `twillSpec`. Seems unnecessary to pass in `twillSpec.getEventHandler()` as a separate parameter.